h/services/orcid_client.py

```diff
@@ -54,7 +61,7 @@ def _get_id_token(self, authorization_code: str) -> str:
 
     def get_orcid(self, authorization_code: str) -> str | None:
         id_token = self._get_id_token(authorization_code)
-        decoded_id_token = JWTService.decode_token(
+        decoded_id_token = self._jwt_service.decode_token(
```

Comment:
`decode_token` is a class method but after this change it is accessed on an instance. This is potentially surprising as it looks like `decode_token` may depend on properties of the instance, but it doesn't.

Reply:
> Would it make sense to change those methods to be instance methods?

Linters sometimes complain if a method takes `self` as an argument but doesn't use it, saying that you should use `@classmethod` or `@staticmethod`. Currently `make lint` doesn't seem to complain about that so I think that must have been a PyLint thing. Nonetheless, I think `@staticmethod` and `@classmethod` still have some readability value in informing the maintainer of `JWTService` that `self` or `cls` isn't intended to be used in this method.